### ISSUE TITLE
Brainstorming on what to add to the docs

### ISSUE THREAD

**Issue by bethesque** [pact-foundation/pact.io#36]
* Where pact fits into the API lifecycle (diagram that @mefellows has)
* Clearer explanation upfront of the process in a language independent way
* Remove unnecessary references to Ruby (though it's a good language for doing code examples in if we do need one, because it is quite readable)
* Value proposition of Pact Broker, the Matrix, can-i-deploy

**Comment by TimothyJones** [comment]
* Repository structure/ecosystem (can be hard to know which repository to report bugs against)
* Improve the general contract testing overview? I think we already have this, but from the StackOverflow questions there seem to be some common misconceptions

**Comment by bethesque** [comment]
* explain the pact specification at a conceptual level and update this with a link: "If you are writing tests on the Consumer side to a different language on the Provider side, you must ensure you use a common Pact Specification between them or you will be unable to validate."

**Comment by bethesque** [comment]
* add info on how to use matchers to their best effect (eg. as lax as possible in the response, stricter in the request)

**Comment by bethesque** [comment]
* Where pact fits into the API lifecycle (diagram that @mefellows has)

**Comment by bethesque** [comment]
* Clearer explanation upfront of the process in a language independent way

**Comment by bethesque** [comment]
* Remove unnecessary references to Ruby (though it's a good language for doing code examples in if we do need one, because it is quite readable)

**Comment by bethesque** [comment]
* Value proposition of Pact Broker, the Matrix, can-i-deploy

**Comment by mboudreau** [comment]
@bethesque Could I suggest that the examples are done in Javascript?  My reasoning being that Javascript is the most well known language and currently our most popular across all our languages.  Since this is a consumer driven contract framework, I'd imagine Javascript would make more sense from the consumer standpoint.  We could potentially keep ruby for the provider side if needed.

**Comment by bethesque** [comment]
Using JS for the examples is a very reasonable suggestion.

**Comment by bethesque** [comment]
* Explain better how the pact is generated

**Comment by TimothyJones** [comment]
* Improve the general introduction to contract testing? I think we already have this, but from the StackOverflow questions there seem to be some common misconceptions

(this is the same as my earlier comment, but I had two points in one comment)

**Comment by mboudreau** [comment]
Yeah, definitely agree.  Currently working on that, but it's slow going
since we have a lot of things to work on and improve.  I need to really
make this a priority over any functional issues since it is problematic.
